tools-local/unix/invoke-extension.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
```

Comment:
Why not fix this now?  IMO things like this should be fixed so that they aren't duplicated when you work on the PS versions.